### ISSUE TITLE
Simplified Chinese Translation Progress

### ISSUE THREAD

**Issue by QC-L** [reactjs/zh-hans.reactjs.org#4]
翻译前，请仔细阅读 [React-中文文档译文排版指南](https://github.com/reactjs/zh-hans.reactjs.org/wiki/React-%E4%B8%AD%E6%96%87%E6%96%87%E6%A1%A3%E8%AF%91%E6%96%87%E6%8E%92%E7%89%88%E6%8C%87%E5%8D%97)。

## For New Translators

To translate a page:

1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. **Take only one page at a time**.
3. Clone this repo, translate your page, and submit a pull request!

Please check the previous fork (https://react.docschina.org/) before starting a translation. Make sure to compare the content with the current React website to makes sure everything is up to date!

## For Maintainers

When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

* [ ] Home Page (@tesseralis) #1

When PRs are merged, make sure to mark that page as completed!

## Core Pages

To do before releasing as an "official" translation. **Please translate these pages first.**

* [x] Tutorial(@yuqingc) #12 
* [x] Home Page(@QC-L)(direct commit)
* [x] Basics
  * [x] Hello World(@QC-L) #9 
  * [x] Introducing JSX(@haimengzhang) #15 
  * [x] Rendering Elements(@robertying) #40 
  * [x] Components and Props(@QC-L) #79 
  * [x] State and Lifecycle(@chenjie3323) #14 
  * [x] Handling Events(@Ryanhui) #84 
  * [x] Conditional Rendering(@MrPluto) #92 
  * [x] Lists and Keys(@zhaozhiming) #41 
  * [x] Forms(@zhangxd6) #95 
  * [x] Lifting State Up(@baxtergu) #90 
  * [x] Composition vs Inheritance(@robertying) #13 
  * [x] Thinking in React (@robertying) #8 
* [x] API Reference
  * [x] React (@baxtergu) #138 
  * [x] React.Component (@musclejack) #164 
  * [x] ReactDOM (@Ryanhui) #96 
  * [x] ReactDOMServer (@pampang) #100 
  * [x] DOM Elements (@zbc) #105 
  * [x] SyntheticEvent (@Ryanhui) #101 
  * [x] Test Utilities (@caiyongmin) #10 
  * [x] Shallow Renderer (@Ryanhui) #91 
  * [x] Test Renderer (@ChloeLiang) #97
  * [x] JS Environment Requirements (@Ryanhui) #88 
  * [x] Glossary (@dear-lizhihua) #155 

### Navigation

These are the navigation links that appears in the sidebar. Possibly wait until
the corresponding sections are translated to do these.

* [x] tutorial/nav.yml (@yuqingc) #12 
* [x] docs/nav.yml (@QC-L) (direct commit)

## Next Steps

These are the next most important translations, ordered by importance:

### Installation

* [x] Getting Started(@laozhu) #143 
* [x] Add React to a Website (@seognil) #103 
* [x] Create a New React App (@seognil) #104 
* [x] CDN Links (@QC-L) #7 

### Hooks

Everyone is hooked on hooks! People are really excited for these and it would be great for them to be translated.

  * [x] Introducing Hooks(@lixiaoyang1992) #131 
  * [x] Hooks at a Glance (@yesmeck) #110 
  * [x] Using the State Hook(@Koopos) #16 
  * [x] Using the Effect Hook (@Crazydogs) #123 
  * [x] Rules of Hooks (@Crazydogs) #135 
  * [x] Building Your Own Hooks (@bailnl) #99 
  * [x] Hooks API Reference (@Yuiham) #173 
  * [x] Hooks FAQ (@tonghuashuo) #121 

### Warnings

These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
 
* [x] dont-call-proptypes (@seognil) #115 
* [x] invalid-aria-prop (@seognil) #114 
* [x] legacy-factories (@seognil) #113 
* [x] invalid-hook-call-warning (@seognil) #169 
* [x] refs-must-have-owner (@seognil) #124
* [x] special-props (@seognil) #112 
* [x] unknown-prop (@seognil) #111 

### Advanced Guides

* [x] Accessibility (@ghycy15) #175
* [x] Code-Splitting (@BetaMee) #11 
* [x] Context (@BetaMee) #87 
* [x] Error Boundaries (@LCINA) #153 
* [x] Forwarding Refs (@yhsunshining) #118 
* [x] Fragments (@chenshuai2144) #132
* [x] Higher-Order Components (@sang4lv) #170  
* [x] Integrating with Other Libraries (@postor) #129
* [x] JSX In Depth (@hstarorg) #158 
* [x] Optimizing Performance (@zombieJ) #120 
* [x] Portals (@zbc) #147 
* [x] React Without ES6 (@zhaozhiming) #128 
* [x] React Without JSX (@Fonkie) #177 
* [x] Reconciliation (@sang4lv) #122 
* [x] Refs and the DOM (@pampang) #154 
* [x] Render Props (@chenshuai2144) #106 
* [x] Static Type Checking (@ycjcl868) #116 
* [x] Strict Mode (@Fonkie)  #165 
* [x] Typechecking With PropTypes (@ycjcl868) #108  
* [x] Uncontrolled Components (@tli4) #119 
* [x] Web Components (@bailnl) #156 

### FAQ

* [x] APIs and AJAX (@hateonion) #157 
* [x] Babel, JSX, and Build Steps (@baxtergu) #167 
* [x] Passing Functions to Components (@sichenguo) #136 
* [x] Component State (@bbbbx) #137 
* [x] Styling and CSS (@infinnie) #134 
* [x] File Structure (@baxtergu) #148 
* [x] Versioning Policy (@buffaybu) #139 
* [x] Virtual DOM and Internals (@ChiuMungZitAlexander) #141 

### Contributing

Priority: Low

* [x] How to Contribute (@jimczj)
* [x] Codebase Overview (@nanaSun) #144 
* [x] Implementation Notes (@nanaSun)
* [x] Design Principles (@tanghejun) #145 

### Components

Components in `src/components` that have some text in them.

* [x] CodeEditor (@zcfan) #149 
* [x] CodeExample (@zcfan) #150 
* [x] ErrorDecoder (@CoolRice) #152 
* [x] LayoutFooter (@QC-L)(direct commit)
* [x] LayoutHeader (@QC-L)(direct commit)
* [x] MarkdownPage (@QC-L)(direct commit)

## Additional Translations

These are not the primary translation targets.

* [ ] Blog
  * [ ] 2013-06-02-jsfiddle-integration
  * [ ] 2013-06-05-why-react (@laobeijing0806) #179
  * [ ] 2013-06-12-community-roundup
  * [ ] 2013-06-19-community-roundup-2
  * [ ] 2013-06-21-react-v0-3-3
  * [ ] 2013-06-27-community-roundup-3
  * [ ] 2013-07-02-react-v0-4-autobind-by-default
  * [ ] 2013-07-03-community-roundup-4
  * [ ] 2013-07-11-react-v0-4-prop-validation-and-default-values
  * [ ] 2013-07-17-react-v0-4-0
  * [ ] 2013-07-23-community-roundup-5
  * [ ] 2013-07-26-react-v0-4-1
  * [ ] 2013-07-30-use-react-and-jsx-in-ruby-on-rails
  * [ ] 2013-08-05-community-roundup-6
  * [ ] 2013-08-19-use-react-and-jsx-in-python-applications (@CalElFe) #171 
  * [ ] 2013-08-26-community-roundup-7
  * [ ] 2013-09-24-community-roundup-8
  * [ ] 2013-10-3-community-roundup-9
  * [ ] 2013-10-16-react-v0.5.0
  * [ ] 2013-10-29-react-v0-5-1
  * [ ] 2013-11-06-community-roundup-10
  * [ ] 2013-11-18-community-roundup-11
  * [ ] 2013-12-18-react-v0.5.2-v0.4.2
  * [ ] 2013-12-19-react-v0.8.0
  * [ ] 2013-12-23-community-roundup-12
  * [ ] 2013-12-30-community-roundup-13
  * [ ] 2014-01-02-react-chrome-developer-tools
  * [ ] 2014-01-06-community-roundup-14
  * [ ] 2014-02-05-community-roundup-15
  * [ ] 2014-02-15-community-roundup-16
  * [ ] 2014-02-16-react-v0.9-rc1
  * [ ] 2014-02-20-react-v0.9
  * [ ] 2014-02-24-community-roundup-17
  * [ ] 2014-03-14-community-roundup-18
  * [ ] 2014-03-19-react-v0.10-rc1
  * [ ] 2014-03-21-react-v0.10
  * [ ] 2014-03-28-the-road-to-1.0
  * [ ] 2014-04-04-reactnet
  * [ ] 2014-05-06-flux
  * [ ] 2014-05-29-one-year-of-open-source-react
  * [ ] 2014-06-27-community-roundup-19
  * [ ] 2014-07-13-react-v0.11-rc1
  * [ ] 2014-07-17-react-v0.11
  * [ ] 2014-07-25-react-v0.11.1
  * [ ] 2014-07-28-community-roundup-20
  * [ ] 2014-07-30-flux-actions-and-the-dispatcher
  * [ ] 2014-08-03-community-roundup-21
  * [ ] 2014-09-03-introducing-the-jsx-specification
  * [ ] 2014-09-12-community-round-up-22
  * [ ] 2014-09-16-react-v0.11.2
  * [ ] 2014-09-24-testing-flux-applications
  * [ ] 2014-10-14-introducing-react-elements
  * [ ] 2014-10-16-react-v0.12-rc1
  * [ ] 2014-10-17-community-roundup-23
  * [ ] 2014-10-27-react-js-conf
  * [ ] 2014-10-28-react-v0.12
  * [ ] 2014-11-24-react-js-conf-updates
  * [ ] 2014-11-25-community-roundup-24
  * [ ] 2014-12-18-react-v0.12.2
  * [ ] 2014-12-19-react-js-conf-diversity-scholarship
  * [ ] 2015-01-27-react-v0.13.0-beta-1
  * [ ] 2015-02-18-react-conf-roundup-2015
  * [ ] 2015-02-20-introducing-relay-and-graphql
  * [ ] 2015-02-24-react-v0.13-rc1
  * [ ] 2015-02-24-streamlining-react-elements
  * [ ] 2015-03-03-react-v0.13-rc2
  * [ ] 2015-03-04-community-roundup-25
  * [ ] 2015-03-10-react-v0.13
  * [ ] 2015-03-16-react-v0.13.1
  * [ ] 2015-03-19-building-the-facebook-news-feed-with-relay
  * [ ] 2015-03-26-introducing-react-native
  * [ ] 2015-03-30-community-roundup-26
  * [ ] 2015-04-17-react-native-v0.4
  * [ ] 2015-04-18-react-v0.13.2
  * [ ] 2015-05-01-graphql-introduction
  * [ ] 2015-05-08-react-v0.13.3
  * [ ] 2015-05-22-react-native-release-process
  * [ ] 2015-06-12-deprecating-jstransform-and-react-tools
  * [ ] 2015-07-03-react-v0.14-beta-1
  * [ ] 2015-08-03-new-react-devtools-beta
  * [ ] 2015-08-11-relay-technical-preview
  * [ ] 2015-08-13-reacteurope-roundup
  * [ ] 2015-09-02-new-react-developer-tools
  * [ ] 2015-09-10-react-v0.14-rc1
  * [ ] 2015-09-14-community-roundup-27
  * [ ] 2015-10-01-react-render-and-top-level-api
  * [ ] 2015-10-07-react-v0.14
  * [ ] 2015-10-19-reactiflux-is-moving-to-discord
  * [ ] 2015-10-28-react-v0.14.1
  * [ ] 2015-11-02-react-v0.14.2
  * [ ] 2015-11-18-react-v0.14.3
  * [ ] 2015-12-04-react-js-conf-2016-diversity-scholarship
  * [ ] 2015-12-16-ismounted-antipattern
  * [ ] 2015-12-18-react-components-elements-and-instances
  * [ ] 2015-12-29-react-v0.14.4
  * [ ] 2016-01-08-A-implies-B-does-not-imply-B-implies-A
  * [ ] 2016-01-12-discontinuing-ie8-support
  * [ ] 2016-02-19-new-versioning-scheme
  * [ ] 2016-03-07-react-v15-rc1
  * [ ] 2016-03-16-react-v15-rc2
  * [ ] 2016-03-29-react-v0.14.8
  * [ ] 2016-04-07-react-v15
  * [ ] 2016-04-08-react-v15.0.1
  * [ ] 2016-07-11-introducing-reacts-error-code-system
  * [ ] 2016-07-13-mixins-considered-harmful
  * [ ] 2016-07-22-create-apps-with-no-configuration
  * [ ] 2016-08-05-relay-state-of-the-state
  * [ ] 2016-09-28-our-first-50000-stars
  * [ ] 2016-11-16-react-v15.4.0
  * [ ] 2017-04-07-react-v15.5.0
  * [ ] 2017-05-18-whats-new-in-create-react-app
  * [ ] 2017-06-13-react-v15.6.0
  * [ ] 2017-07-26-error-handling-in-react-16
  * [ ] 2017-09-08-dom-attributes-in-react-16
  * [ ] 2017-09-25-react-v15.6.2
  * [ ] 2017-09-26-react-v16.0
  * [ ] 2017-11-28-react-v16.2.0-fragment-support
  * [ ] 2017-12-07-introducing-the-react-rfc-process
  * [ ] 2017-12-15-improving-the-repository-infrastructure
  * [ ] 2018-03-01-sneak-peek-beyond-react-16
  * [ ] 2018-03-27-update-on-async-rendering(@Fonkie) #282 
  * [ ] 2018-03-29-react-v-16-3(@Fonkie) 
  * [ ] 2018-05-23-react-v-16-4
  * [ ] 2018-06-07-you-probably-dont-need-derived-state (@maicss) #168
  * [ ] 2018-08-01-react-v-16-4-2
  * [ ] 2018-09-10-introducing-the-react-profiler(@Fonkie) #253 
  * [ ] 2018-10-01-create-react-app-v2 (@hstarorg)
  * [ ] 2018-10-23-react-v-16-6
  * [ ] 2018-11-13-react-conf-recap (@manglobe) #181 
  * [ ] 2018-11-27-react-16-roadmap (@ghycy15)
  * [ ] 2018-12-19-react-v-16-7
  * [ ] 2019-02-06-react-v16.8.0 (@JiyuShao) #171 
  * [ ] 2019-02-23-is-react-translated-yet (@penghk)
  * [ ] 2020-02-26-react-v16.13.0 (@Yilia-Wish)
* [ ] Community

**Comment by robertying** [comment]
I would like to translate, starting with “Thinking in React” if that’s ok.

But we need to wait for the glossary to be finished first, don’t we? Or is there anything I can help with too?

**Comment by tesseralis** [comment]
@robertying I think you can start. The previous version of the glossary is here for reference: https://react.docschina.org/docs/glossary.html

Remember to reference the previous version of the translation (https://react.docschina.org/docs/thinking-in-react.html) and just make sure it's accurate to the current docs (https://reactjs.org/docs/thinking-in-react.html

**Comment by robertying** [comment]
@tesseralis their original work seems more than intact. Translating those pages would be mostly reviewing instead of editing. I don’t tend to take their credit for those. Maybe I should start with something untranslated like pages about hooks. What do you think?

**Comment by tesseralis** [comment]
Sure! I think some of the hooks pages are already translated but you're free to pick one that isn't :)

**Comment by robertying** [comment]
I’ll start from bottom to top so the first will be “Hooks FAQ” 😊

**Comment by tesseralis** [comment]
@QC-L are you and the team able to handle porting over the rest of the Chinese site in the next few weeks? Would you like folks like @robertying to help?

**Comment by QC-L** [comment]
@tesseralis  I will give the translation process and specifications.

**Comment by QC-L** [comment]
@robertying You can choose one, if you think it is not good before, you can re-translate.

**Comment by QC-L** [comment]
@robertying The previous content did not have some review, this time will pay attention, thank you for reminding.
Thank you for your help, make this repo perfect.

**Comment by lcxfs1991** [comment]
Happy to see your enthusiasm @robertying. There seems to be some defects in old version because reviewer did not review in time. Though we should be thankful since a lot of people voluntarily help the translation. Open source community needs both hero and the mass.

Looking forward to your contribution!

**Comment by QC-L** [comment]
@robertying Past content can be referenced, but re-translation is best.

**Comment by robertying** [comment]
Cool, thanks for the suggestions. 😀 After careful considerations, I think I'd better start to review the old ones first so the website can get working soon. As was mentioned, I'll start from bottom to top. The first would be reviewing `Thinking in React`.

**Comment by QC-L** [comment]
Yeah, that's great. 
I will add the QR code of the WeChat group in README.md.
You can join us, as @lcxfs1991 says the community need hero.

**Comment by yuqingc** [comment]
I'd like to claim the "tutorial" part if it is not taken yet.

**Comment by caiyongmin** [comment]
I'd like to translate the `Test Utilities` part,
if it is not taken yet.

**Comment by MrPluto** [comment]
Pleasure to translate `Conditional Rendering`

**Comment by baxtergu** [comment]
Would like to take `Lifting State Up` as a start.

**Comment by chenjie3323** [comment]
Want to start at `State and Lifecycle`.

**Comment by robertying** [comment]
Continue with `Composition vs Inheritance`, if it's ok

**Comment by QC-L** [comment]
@haimengzhang You can translate it.

**Comment by Koopos** [comment]
anyone working on using state hooks?

**Comment by QC-L** [comment]
@Koopos We focus on `Basics`.

**Comment by robertying** [comment]
I'll continue to work on `Rendering Elements`.

**Comment by hijiangtao** [comment]
We may need a better structure to organize the translation list, it's pretty difficult to see the progress of it, currently.

For example, use **table** rather than list:

|Title|Processing|Status|
|---|---|---|
|Hello World|(@ GitHub ID here)|Translating/Proofreading, etc.|

**Comment by Ryanhui** [comment]
I will translate `Handling Events`

**Comment by zhangxd6** [comment]
I would like to try the form if it is not taken

**Comment by BetaMee** [comment]
| Title | Processing | Status |
|---|---|---|
|Context| @BetaMee | Translating |

**Comment by Ryanhui** [comment]
| Title      | Pocessing     | Status     |
| :------------- | :----------: | -----------: |
| API Reference/JS Environment Requirements | @Ryanhui    | Translating    |

**Comment by Ryanhui** [comment]
|Title|Pocessing|Status|
|--------|:-------------:|------:|
|API Reference/Shallow Renderer| @Ryanhui |Translating|

**Comment by baxtergu** [comment]
* Application Status: Waiting for approve from @QC-L 

| Title | Processing | Status |
| - | - | - |
| API Reference / React | @baxtergu | Translating |
| API Reference / React.Component | @baxtergu | Translating |

**Comment by Ryanhui** [comment]
|Title|Processing|Status|
|-----|-----|----|
|API Reference / ReactDOM|@Ryanhui|Translating|

**Comment by laozhu** [comment]
|                     Title                       | Processing |     Status     |
|                      :---:                      |      :---:        |      :---:       |
| Installation /  Getting Started  |   @laozhu   | Translating |

**Comment by lixiaoyang1992** [comment]
|                     Title                       | Processing |     Status     |
|                      :---:                      |      :---:        |      :---:       |
|Hooks /  Introducing Hooks|   @lixiaoyang1992   | Translating |

**Comment by zbc** [comment]
| Title        | Processing           | Status  |
| :-------------: |:-------------:| :-----:|
|  API Reference / DOM Elements   | @zbc | Translating |

**Comment by ChloeLiang** [comment]
I would like to translate Test Renderer.

**Comment by tesseralis** [comment]
Could one of the maintainers mark which pages have pull requests for them? e.g.

* [ ] Tutorial (@yuqingc) #12

**Comment by QC-L** [comment]
@tesseralis OK, I modifying it.

**Comment by Ryanhui** [comment]
|Title|Processing|Status|
|----|----|----|
|API Reference / SyntheticEvent| @Ryanhui | Translating |

**Comment by haimengzhang** [comment]
I can take React.Component if no one is

**Comment by bailnl** [comment]
I can take **Building Your Own Hooks**, if no one is

**Comment by pampang** [comment]
I would like to take **API Reference / ReactDOMServer**, if it is not taken.

**Comment by yesmeck** [comment]
I'd like to take **Hooks / Hooks at a Glance**.

**Comment by chenshuai2144** [comment]
I'd like to take  **Render Props** and **Static Type Checking**

**Comment by ycjcl868** [comment]
I'd link to take __Static Type Checking__ and __Typechecking With PropTypes__

**Comment by seognil** [comment]
| Title | Processing | Status |
| :------: | :------: | :------: |
| Installation / Add React to a Website | @seognil | Translating |
| Installation / Create a New React App | @seognil | Translating |

**Comment by hijiangtao** [comment]
Thanks for all of your enthusiastic participation in React Simplified Chinese Documents, and let's follow current guidance (given in the beginning by @QC-L) - **Take only one page at a time**, to make the translation progress more consistent and organized. 

You can claim again after a PR is raised for your current translation. Currently, we regard the first article in each comment as your current translation claim.

You are also welcomed to give your estimated time or date for your translation claims, such as:

> EST.cmplt Date: 2019.02.14

or 

> EST.cmplt Time: within 2 days

Then we can keep an eye on it and give you helps when it's necessary.

**Comment by Crazydogs** [comment]
I'd like to take **Using the Effect Hook** and **Rules of Hooks**

| Title   |      Processing      |  Status |
|----------|:-------------:|------:|
| Hooks/ Using the Effect Hook |  @Crazydogs | Translating |
| Hooks/  Rules of Hooks |  @Crazydogs | Translating |

**Comment by Yuiham** [comment]
I'd like to take **Hooks API Reference**

| Title | Processing | Status |
| --- | --- | --- |
| Hooks/ Hooks API Reference | @Yuiham | Translating |

**Comment by tonghuashuo** [comment]
I'd like to take **Hooks/Hooks FAQ**

| Title | Processing | Status |
|-|-|-|
| Hooks/Hooks FAQ | @tonghuashuo | Translating |

**Comment by zombieJ** [comment]
volunteer for **Using the Effect Hook**

**Comment by dear-lizhihua** [comment]
I'd like to take **Glossary of React Terms**
https://reactjs.org/docs/glossary.html

**Comment by seognil** [comment]
| Title | Processing | Status |
| :------: | :------: | :------: |
| warnings/invalid-aria-prop | @seognil | Translating  |
| warnings/legacy-factories | @seognil | Translating  |
| warnings/special-props | @seognil | Translating  |

I'd use permalink from `.md` for warning section instead
(these three are shortest LOL

**Comment by yhsunshining** [comment]
could I take **Advanced Guides/Forwarding Refs**?

|  Title|Processing|status|
|---|---|---|
| Advanced Guides/Forwarding Refs | @yhsunshining |　Translating|

**Comment by seognil** [comment]
| Title | Processing | Status |
| :------: | :------: | :------: |
| warnings/dont-call-proptypes | @seognil | #115 |
| warnings/invalid-hook-call-warning | @seognil | Translating |
| warnings/refs-must-have-owner | @seognil | Translating |
| warnings/unknown-prop | @seognil | #111 |

**Comment by LCINA** [comment]
I'd like to take **Error Boundaries**
Title | Processing | Status
------------ | ------------- | -------------
Error Boundaries | LCINA | Translating

**Comment by JIACHENG9** [comment]
Title | Processing | Status
------------ | ------------- | -------------
JSX In Depth | @JIACHENG9 | Translating

**Comment by JIACHENG9** [comment]
Title | Processing | Status
------------ | ------------- | -------------
React Without JSX | @JIACHENG9 | Translating

**Comment by tli4** [comment]
Title | Processing | Status
------------ | ------------- | -------------
Advanced Guides / Uncontrolled Components | @tli4  | Translating

**Comment by summccc** [comment]
I'd like to take **Accessibility** please.

**Comment by sang4lv** [comment]
| Title |	Processing | 	Status |
| Reconciliation | @sang4lv |	Translating |

**Comment by bailnl** [comment]
I'd like to take **Reconciliation** ， if no one is.

**Comment by zbc** [comment]
|Title|Processing |	Status
|:----:|:----:|:----:|
|Portals	|@zbc|	Translating

**Comment by sang4lv** [comment]
I'll take HOC and Portals

| Title |	Processing | 	Status |
| - | - | - |
| Higher-Order Components | @sang4lv |	Translating |
| Portals | @sang4lv |	Translating |

**Comment by pampang** [comment]
I would like to take **Refs and the DOM**, if it is not taken.

**Comment by zhaozhiming** [comment]
I would like to take **React Without ES6**, if it is not taken.

**Comment by postor** [comment]
I'll take  **Integrating with Other Libraries**

<table>
<thead>
<tr>
<th>Title</th>
<th>Processing</th>
<th>Status</th>
</tr>
</thead>
<tbody>
<tr>
<td>Integrating with Other Libraries</td>
<td>
@postor
</td>
<td>Translating</td>
</tr>
</tbody>
</table>

**Comment by Fonkie** [comment]
I'll take **Strict Mode**

Title | Processing | Status
---|:--:|---:
Strict Mode | @Fonkie | Translating

**Comment by nanaSun** [comment]
I'll take *** Codebase Overview**

|Title	|Processing	|Status
:--:|:--:|:--:
 Codebase  Overview	|@nanaSun| Translating

**Comment by hateonion** [comment]
|Title|Processing|Status|
|----|----|----|
|AJAX and APIs|@hateonion|Translating|

**Comment by QC-L** [comment]
I'll take ** Babel, JSX, and Build Steps **

|Title	|Processing	|Status|
|:--:|:--:|:--: |
|Babel, JSX, and Build Step	| @QC-L | Translating|

**Comment by bbbbx** [comment]
I volunteer to translate:

| Title | Processing | Status |
|:-----|:-----------|:-------|
| Component State | @bbbbx | Translating |

**Comment by infinnie** [comment]
I take:
---
id: faq-styling
title: Styling and CSS
permalink: docs/faq-styling.html
layout: docs
category: FAQ
---

**Comment by sichenguo** [comment]
> could I take *** FAQ/Passing Functions to Components**?
> 
> Title	Processing	status
> Passing Functions to Components	@sichenguo	　Translating

**Comment by sichenguo** [comment]
> could I take *** FAQ/Passing Functions to Components**?
> 
> Title	Processing	status
> Passing Functions to Components	@sichenguo	　Translating

**Comment by sichenguo** [comment]
could I take **FAQ/Passing Functions to Components**?
 
| Title | Processing |status
| ------ | ------ | ------ |
 |Passing Functions to Components	|@sichenguo	　|Translating

**Comment by buffaybu** [comment]
I'd like to take **Versioning Policy**.

| Title  | Processing | Status |
| --- | --- | --- |
| Versioning Policy  | @buffaybu  | Translating

**Comment by ChiuMungZitAlexander** [comment]
Looks like there're only a few left. So how about I take **Virtual DOM and Internals**?

| **Title** | **Processing** | **Status** |
| ------ | ------ | ------ |
| Virtual DOM and Internals | @ChiuMungZitAlexander| Translating |

**Comment by baxtergu** [comment]
| Title | Processing | Status |
| - | - | - |
| FAQ/ File Structure| @baxtergu | Translating  |

**Comment by tanghejun** [comment]
| Title        | Processing          | Status  |
| ------------- |:-------------:| -----:|
|  Contributing/Design Principles | @tanghejun | Translating |

**Comment by CoolRice** [comment]
Title	Processing	Status
Components/ErrorDecoder	@CoolRice	Translating

**Comment by musclejack** [comment]
|Title|Processing|Status|
 |:----:|:----|----:|
 |React.Component|@musclejack|Translating|

**Comment by zcfan** [comment]
I'll work on "Hooks state" since @Koopos is inactive.

Title|Processing|Status
----|----|----
Hooks/ Using the State Hook|@zcfan|Translating

**Comment by QC-L** [comment]
@zcfan NO, you can review this PR #16

**Comment by jimczj** [comment]
| Title |	Processing |	Status
|---------- |--------- |---------- |
| Contributing/ How to Contribute | @jimczj | Translating

**Comment by nanaSun** [comment]
I'll take **Implementation Notes**
|Title	|Processing	|Status
:--:|:--:|:--:
Implementation Notes | @nanaSun | Translating

**Comment by Runbo94** [comment]
> I'll take Implementation Notes
> 
> 
> 
> Title
> Processing
> Status
> 
> 
> 
> 
> Implementation Notes
> @nanaSun
> Translating

你翻译多少了。。。我翻译到 Updating{#updating}了

**Comment by QC-L** [comment]
@Runbo94 下次请领取任务之后再提交内容，谢谢合作😝

**Comment by zcfan** [comment]
Title | Processing | Status
--|--|--
CodeEditor | @zcfan | Translating

**Comment by zcfan** [comment]
Title | Processing | Status
--|--|--
CodeExample | @zcfan | Translating

**Comment by YaokaiYang-assaultmaster** [comment]
Hey there, looks like all the parts have been taken. Are there any sections I can help about? Please let me know and I am definitely willing to help. @QC-L

**Comment by QC-L** [comment]
@YaokaiYang-assaultmaster That's great. You can review the translated articles.

**Comment by hstarorg** [comment]
| Title | Processing | Status |
|---|---|---|
|  jsx-in-depth | @hstarorg  | Translating  |

**Comment by maicss** [comment]
> Title	Processing	Status
> React.Component	@musclejack	Translating

hi，你翻译多少啦？

**Comment by QC-L** [comment]
@maicss

**Comment by CalElFe** [comment]
Hi, can I take any of blog pages? It seems like all the "Primary" tasks are already taken.

**Comment by QC-L** [comment]
@CalElFe Yeah, that's awesome. I'll add blog related content. You can try.

**Comment by maicss** [comment]
@QC-L 我想问一下，如果英文的文档更新了，那到时候怎么记录差异，然后更新中文文档呢？

**Comment by maicss** [comment]
|Titel|Processing|Status|
|:--|:--|:--|
|blog/You Probably Don't Need Derived State|@maicss|Translating|
|blog/Introducing Hooks|@maicss|Translating|

**Comment by QC-L** [comment]
@maicss 差异问题，我会解决。

**Comment by penghk** [comment]
I'd like to claim the blog "2019-02-23-is-react-translated-yet" if it is not taken yet.

**Comment by JiyuShao** [comment]
Can I take the "2019-02-06-react-v16.8.0" page?

Title | Processing | Status 
-- | -- | --
2019-02-06-react-v16.8.0 | @JiyuShao | Translating

**Comment by ghycy15** [comment]
Can I take "2018-11-27-react-16-roadmap"?

**Comment by ghycy15** [comment]
@summccc I have used Accessibility features recently. I can help if you are facing any difficulties.

**Comment by hstarorg** [comment]
| Title | Processing | Status | 
| --- | --- | --- |
| blog/2018-10-01-create-react-app-v2.md | @hstarorg | Translating |

**Comment by CalElFe** [comment]
Can I take "2013-08-19-use-react-and-jsx-in-python-applications"?

| **Title**| **Processing**|**Status**|
|-----------|----------------|--------------|
|2013-08-19-use-react-and-jsx-in-python-applications| @CalElFe |Translating|

**Comment by ghycy15** [comment]
Can I also take Accessibility? I have a lot passion in that topic~
| Title |Processing| Status |
| Accessibility |@ghycy15 |Translating |

**Comment by lvdingjin** [comment]
Pleasure to translate the rest of "Using the Effect Hook", if it is ok.

**Comment by QC-L** [comment]
@lvdingjin You can review #123, it has been translated

**Comment by Cygra** [comment]
| Title | Processing | Status
| --- | --- | ---|
| Hooks API Reference | @Cygra | Translating |

**Comment by QC-L** [comment]
@Cygra Already someone is translating.

**Comment by Cygra** [comment]
> @Cygra Sorry, it's already someone is translating. You can try blog.

ooops

**Comment by tesseralis** [comment]
恭喜!

<img width="362" alt="Screen Shot 2019-03-20 at 12 46 06 PM" src="https://user-images.githubusercontent.com/1278991/54725679-38cbc280-4b2d-11e9-867b-b29c94304001.png">

**Comment by ghycy15** [comment]
Title | Processing | Status
-- | -- | --
Accessibility | @ghycy15 | 175

**Comment by ghycy15** [comment]
Is there any plan to translate react native? https://facebook.github.io/react-native/

**Comment by laobeijing0806** [comment]
Title | Processing | Status
-- | -- | --
blog/2013-06-05-why-react | @laobeijing0806 | Translating

**Comment by QC-L** [comment]
@ghycy15 I think the priority now should be to keep React well.

**Comment by hstarorg** [comment]
| Title | Processing | Status |
|---|---|---|
| docs/legacy-context.md | @hstarorg  | Translating |

**Comment by manglobe** [comment]
|Title|Processing|Status|
|---|---|---|
|blog/2018-11-13-react-conf-recap| @manglobe | Translating

**Comment by manglobe** [comment]
|Title|Processing|Status|
|---|---|---|
|blog/2018-10-23-react-v-16-6| @manglobe | Translating

**Comment by Fonkie** [comment]
Title | Processing | Status
-- | -- | --
blog/2018-09-10-introducing-the-react-profiler | @Fonkie  | Translating

**Comment by rdai10** [comment]
I would like to help translate blog/2018-12-19-react-v-16-7 if no one has started on it.

**Comment by amnsss** [comment]
I would like to help translate blog/2018-03-27-update-on-async-rendering if no one has started on it.


Title | Processing | Status
-- | -- | --
 blog/2018-03-27-update-on-async-rendering | @amnsss | Translating

**Comment by ghycy15** [comment]
Title | Processing | Status
-- | -- | --
blog/2018-11-27-react-16-roadmap | @ghycy15 | #234

**Comment by jialin-wu-02** [comment]
Start woking on the translation of blog/2016-09-28-our-first-50000-stars. Tbh it's quite a exciting one lolol.

**Comment by matrixqc** [comment]
advanced guides -> [profiler API](https://zh-hans.reactjs.org/docs/profiler.html) is not in catalog.

**Comment by QC-L** [comment]
@matrixqc you can do it

**Comment by matrixqc** [comment]
Title | Processing |  Status  
-|-|-
Advanced Guides/Profiler API | @matrixqc  | Translating |

**Comment by Rattlingshinuiba** [comment]
I'd like to claim the `How to Contribute` part if it is not taken yet.

**Comment by QC-L** [comment]
@Rattlingshinuiba That article has been translated.

**Comment by Rattlingshinuiba** [comment]
@QC-L 谢谢提醒！对不起这几天没好好关注动态，不过我目前无论从中文官网上还是从中文库上，都没发现我所提及文章的中文版本，之后我搜了下 issue，发现了一个 closed 的同名文章 issue，麻烦你百忙中请核对一下。

**Comment by QC-L** [comment]
@Rattlingshinuiba okey

**Comment by zcfan** [comment]
is 2017-12-15-improving-the-repository-infrastructure taken?


Title | Processing | Status 
-- | -- | --
2017-12-15-improving-the-repository-infrastructure| @zcfan | Translating

**Comment by doudounannan** [comment]
is Suspense for Data Fetching taken?

Title | Processing | Status
-- | -- | --
Suspense for Data Fetching | @doudounannan | Translating

**Comment by yes1am** [comment]
| title | Processing | Status |
| --- | --- | --- |
| Legacy Context | @yes1am | done |  

[pull request link](https://github.com/reactjs/zh-hans.reactjs.org/pull/370)

**Comment by TabrisK** [comment]
Title|Processing|Status
--|--|--
2015-12-18-react-components-elements-and-instances|@TabrisK|Translating

**Comment by dylanyg** [comment]
> 翻译前，请仔细阅读 [React-中文文档译文排版指南](https://github.com/reactjs/zh-hans.reactjs.org/wiki/React-%E4%B8%AD%E6%96%87%E6%96%87%E6%A1%A3%E8%AF%91%E6%96%87%E6%8E%92%E7%89%88%E6%8C%87%E5%8D%97)。
> 
> ## For New Translators
> To translate a page:
> 
> 1. Check that no one else has claimed your page in the checklist and comments below.
> 2. Comment below with the name of the page you would like to translate. **Take only one page at a time**.
> 3. Clone this repo, translate your page, and submit a pull request!
> 
> Please check the previous fork (https://react.docschina.org/) before starting a translation. Make sure to compare the content with the current React website to makes sure everything is up to date!
> 
> ## For Maintainers
> When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:
> 
> * [ ]  Home Page (@tesseralis) #1
> 
> When PRs are merged, make sure to mark that page as completed!
> 
> ## Core Pages
> To do before releasing as an "official" translation. **Please translate these pages first.**
> 
> * [x]  Tutorial(@yuqingc) #12
> * [x]  Home Page(@QC-L)(direct commit)
> * [x]  Basics
>   
>   * [x]  Hello World(@QC-L) #9
>   * [x]  Introducing JSX(@haimengzhang) #15
>   * [x]  Rendering Elements(@robertying) #40
>   * [x]  Components and Props(@QC-L) #79
>   * [x]  State and Lifecycle(@chenjie3323) #14
>   * [x]  Handling Events(@Ryanhui) #84
>   * [x]  Conditional Rendering(@MrPluto) #92
>   * [x]  Lists and Keys(@zhaozhiming) #41
>   * [x]  Forms(@zhangxd6) #95
>   * [x]  Lifting State Up(@baxtergu) #90
>   * [x]  Composition vs Inheritance(@robertying) #13
>   * [x]  Thinking in React (@robertying) #8
> * [x]  API Reference
>   
>   * [x]  React (@baxtergu) #138
>   * [x]  React.Component (@musclejack) #164
>   * [x]  ReactDOM (@Ryanhui) #96
>   * [x]  ReactDOMServer (@pampang) #100
>   * [x]  DOM Elements (@zbc) #105
>   * [x]  SyntheticEvent (@Ryanhui) #101
>   * [x]  Test Utilities (@caiyongmin) #10
>   * [x]  Shallow Renderer (@Ryanhui) #91
>   * [x]  Test Renderer (@ChloeLiang) #97
>   * [x]  JS Environment Requirements (@Ryanhui) #88
>   * [x]  Glossary (@dear-lizhihua) #155
> 
> ### Navigation
> These are the navigation links that appears in the sidebar. Possibly wait until
> the corresponding sections are translated to do these.
> 
> * [x]  tutorial/nav.yml (@yuqingc) #12
> * [x]  docs/nav.yml (@QC-L) (direct commit)
> 
> ## Next Steps
> These are the next most important translations, ordered by importance:
> 
> ### Installation
> * [x]  Getting Started(@laozhu) #143
> * [x]  Add React to a Website (@seognil) #103
> * [x]  Create a New React App (@seognil) #104
> * [x]  CDN Links (@QC-L) #7
> 
> ### Hooks
> Everyone is hooked on hooks! People are really excited for these and it would be great for them to be translated.
> 
> * [x]  Introducing Hooks(@lixiaoyang1992) #131
> * [x]  Hooks at a Glance (@yesmeck) #110
> * [x]  Using the State Hook(@Koopos) #16
> * [x]  Using the Effect Hook (@Crazydogs) #123
> * [x]  Rules of Hooks (@Crazydogs) #135
> * [x]  Building Your Own Hooks (@bailnl) #99
> * [x]  Hooks API Reference (@Yuiham) #173
> * [x]  Hooks FAQ (@tonghuashuo) #121
> 
> ### Warnings
> These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
> 
> * [x]  dont-call-proptypes (@seognil) #115
> * [x]  invalid-aria-prop (@seognil) #114
> * [x]  legacy-factories (@seognil) #113
> * [x]  invalid-hook-call-warning (@seognil) #169
> * [x]  refs-must-have-owner (@seognil) #124
> * [x]  special-props (@seognil) #112
> * [x]  unknown-prop (@seognil) #111
> 
> ### Advanced Guides
> * [x]  Accessibility (@ghycy15) #175
> * [x]  Code-Splitting (@BetaMee) #11
> * [x]  Context (@BetaMee) #87
> * [x]  Error Boundaries (@LCINA) #153
> * [x]  Forwarding Refs (@yhsunshining) #118
> * [x]  Fragments (@chenshuai2144) #132
> * [x]  Higher-Order Components (@sang4lv) #170
> * [x]  Integrating with Other Libraries (@postor) #129
> * [x]  JSX In Depth (@hstarorg) #158
> * [x]  Optimizing Performance (@zombieJ) #120
> * [x]  Portals (@zbc) #147
> * [x]  React Without ES6 (@zhaozhiming) #128
> * [x]  React Without JSX (@Fonkie) #177
> * [x]  Reconciliation (@sang4lv) #122
> * [x]  Refs and the DOM (@pampang) #154
> * [x]  Render Props (@chenshuai2144) #106
> * [x]  Static Type Checking (@ycjcl868) #116
> * [x]  Strict Mode (@Fonkie)  #165
> * [x]  Typechecking With PropTypes (@ycjcl868) #108
> * [x]  Uncontrolled Components (@tli4) #119
> * [x]  Web Components (@bailnl) #156
> 
> ### FAQ
> * [x]  APIs and AJAX (@hateonion) #157
> * [x]  Babel, JSX, and Build Steps (@baxtergu) #167
> * [x]  Passing Functions to Components (@sichenguo) #136
> * [x]  Component State (@bbbbx) #137
> * [x]  Styling and CSS (@infinnie) #134
> * [ ]  File Structure (@baxtergu) #148
> * [x]  Versioning Policy (@buffaybu) #139
> * [x]  Virtual DOM and Internals (@ChiuMungZitAlexander) #141
> 
> ### Contributing
> Priority: Low
> 
> * [x]  How to Contribute (@jimczj)
> * [x]  Codebase Overview (@nanaSun) #144
> * [x]  Implementation Notes (@nanaSun)
> * [x]  Design Principles (@tanghejun) #145
> 
> ### Components
> Components in `src/components` that have some text in them.
> 
> * [x]  CodeEditor (@zcfan) #149
> * [x]  CodeExample (@zcfan) #150
> * [x]  ErrorDecoder (@CoolRice) #152
> * [x]  LayoutFooter (@QC-L)(direct commit)
> * [x]  LayoutHeader (@QC-L)(direct commit)
> * [x]  MarkdownPage (@QC-L)(direct commit)
> 
> ## Additional Translations
> These are not the primary translation targets.
> 
> * [ ]  Blog
>   
>   * [ ]  2013-06-02-jsfiddle-integration
>   * [ ]  2013-06-05-why-react (@laobeijing0806) #179
>   * [ ]  2013-06-12-community-roundup
>   * [ ]  2013-06-19-community-roundup-2
>   * [ ]  2013-06-21-react-v0-3-3
>   * [ ]  2013-06-27-community-roundup-3
>   * [ ]  2013-07-02-react-v0-4-autobind-by-default
>   * [ ]  2013-07-03-community-roundup-4
>   * [ ]  2013-07-11-react-v0-4-prop-validation-and-default-values
>   * [ ]  2013-07-17-react-v0-4-0
>   * [ ]  2013-07-23-community-roundup-5
>   * [ ]  2013-07-26-react-v0-4-1
>   * [ ]  2013-07-30-use-react-and-jsx-in-ruby-on-rails
>   * [ ]  2013-08-05-community-roundup-6
>   * [ ]  2013-08-19-use-react-and-jsx-in-python-applications (@CalElFe) #171
>   * [ ]  2013-08-26-community-roundup-7
>   * [ ]  2013-09-24-community-roundup-8
>   * [ ]  2013-10-3-community-roundup-9
>   * [ ]  2013-10-16-react-v0.5.0
>   * [ ]  2013-10-29-react-v0-5-1
>   * [ ]  2013-11-06-community-roundup-10
>   * [ ]  2013-11-18-community-roundup-11
>   * [ ]  2013-12-18-react-v0.5.2-v0.4.2
>   * [ ]  2013-12-19-react-v0.8.0
>   * [ ]  2013-12-23-community-roundup-12
>   * [ ]  2013-12-30-community-roundup-13
>   * [ ]  2014-01-02-react-chrome-developer-tools
>   * [ ]  2014-01-06-community-roundup-14
>   * [ ]  2014-02-05-community-roundup-15
>   * [ ]  2014-02-15-community-roundup-16
>   * [ ]  2014-02-16-react-v0.9-rc1
>   * [ ]  2014-02-20-react-v0.9
>   * [ ]  2014-02-24-community-roundup-17
>   * [ ]  2014-03-14-community-roundup-18
>   * [ ]  2014-03-19-react-v0.10-rc1
>   * [ ]  2014-03-21-react-v0.10
>   * [ ]  2014-03-28-the-road-to-1.0
>   * [ ]  2014-04-04-reactnet
>   * [ ]  2014-05-06-flux
>   * [ ]  2014-05-29-one-year-of-open-source-react
>   * [ ]  2014-06-27-community-roundup-19
>   * [ ]  2014-07-13-react-v0.11-rc1
>   * [ ]  2014-07-17-react-v0.11
>   * [ ]  2014-07-25-react-v0.11.1
>   * [ ]  2014-07-28-community-roundup-20
>   * [ ]  2014-07-30-flux-actions-and-the-dispatcher
>   * [ ]  2014-08-03-community-roundup-21
>   * [ ]  2014-09-03-introducing-the-jsx-specification
>   * [ ]  2014-09-12-community-round-up-22
>   * [ ]  2014-09-16-react-v0.11.2
>   * [ ]  2014-09-24-testing-flux-applications
>   * [ ]  2014-10-14-introducing-react-elements
>   * [ ]  2014-10-16-react-v0.12-rc1
>   * [ ]  2014-10-17-community-roundup-23
>   * [ ]  2014-10-27-react-js-conf
>   * [ ]  2014-10-28-react-v0.12
>   * [ ]  2014-11-24-react-js-conf-updates
>   * [ ]  2014-11-25-community-roundup-24
>   * [ ]  2014-12-18-react-v0.12.2
>   * [ ]  2014-12-19-react-js-conf-diversity-scholarship
>   * [ ]  2015-01-27-react-v0.13.0-beta-1
>   * [ ]  2015-02-18-react-conf-roundup-2015
>   * [ ]  2015-02-20-introducing-relay-and-graphql
>   * [ ]  2015-02-24-react-v0.13-rc1
>   * [ ]  2015-02-24-streamlining-react-elements
>   * [ ]  2015-03-03-react-v0.13-rc2
>   * [ ]  2015-03-04-community-roundup-25
>   * [ ]  2015-03-10-react-v0.13
>   * [ ]  2015-03-16-react-v0.13.1
>   * [ ]  2015-03-19-building-the-facebook-news-feed-with-relay
>   * [ ]  2015-03-26-introducing-react-native
>   * [ ]  2015-03-30-community-roundup-26
>   * [ ]  2015-04-17-react-native-v0.4
>   * [ ]  2015-04-18-react-v0.13.2
>   * [ ]  2015-05-01-graphql-introduction
>   * [ ]  2015-05-08-react-v0.13.3
>   * [ ]  2015-05-22-react-native-release-process
>   * [ ]  2015-06-12-deprecating-jstransform-and-react-tools
>   * [ ]  2015-07-03-react-v0.14-beta-1
>   * [ ]  2015-08-03-new-react-devtools-beta
>   * [ ]  2015-08-11-relay-technical-preview
>   * [ ]  2015-08-13-reacteurope-roundup
>   * [ ]  2015-09-02-new-react-developer-tools
>   * [ ]  2015-09-10-react-v0.14-rc1
>   * [ ]  2015-09-14-community-roundup-27
>   * [ ]  2015-10-01-react-render-and-top-level-api
>   * [ ]  2015-10-07-react-v0.14
>   * [ ]  2015-10-19-reactiflux-is-moving-to-discord
>   * [ ]  2015-10-28-react-v0.14.1
>   * [ ]  2015-11-02-react-v0.14.2
>   * [ ]  2015-11-18-react-v0.14.3
>   * [ ]  2015-12-04-react-js-conf-2016-diversity-scholarship
>   * [ ]  2015-12-16-ismounted-antipattern
>   * [ ]  2015-12-18-react-components-elements-and-instances
>   * [ ]  2015-12-29-react-v0.14.4
>   * [ ]  2016-01-08-A-implies-B-does-not-imply-B-implies-A
>   * [ ]  2016-01-12-discontinuing-ie8-support
>   * [ ]  2016-02-19-new-versioning-scheme
>   * [ ]  2016-03-07-react-v15-rc1
>   * [ ]  2016-03-16-react-v15-rc2
>   * [ ]  2016-03-29-react-v0.14.8
>   * [ ]  2016-04-07-react-v15
>   * [ ]  2016-04-08-react-v15.0.1
>   * [ ]  2016-07-11-introducing-reacts-error-code-system
>   * [ ]  2016-07-13-mixins-considered-harmful
>   * [ ]  2016-07-22-create-apps-with-no-configuration
>   * [ ]  2016-08-05-relay-state-of-the-state
>   * [ ]  2016-09-28-our-first-50000-stars
>   * [ ]  2016-11-16-react-v15.4.0
>   * [ ]  2017-04-07-react-v15.5.0
>   * [ ]  2017-05-18-whats-new-in-create-react-app
>   * [ ]  2017-06-13-react-v15.6.0
>   * [ ]  2017-07-26-error-handling-in-react-16
>   * [ ]  2017-09-08-dom-attributes-in-react-16
>   * [ ]  2017-09-25-react-v15.6.2
>   * [ ]  2017-09-26-react-v16.0
>   * [ ]  2017-11-28-react-v16.2.0-fragment-support
>   * [ ]  2017-12-07-introducing-the-react-rfc-process
>   * [ ]  2017-12-15-improving-the-repository-infrastructure
>   * [ ]  2018-03-01-sneak-peek-beyond-react-16
>   * [ ]  2018-03-27-update-on-async-rendering(@Fonkie) #282
>   * [ ]  2018-03-29-react-v-16-3(@Fonkie)
>   * [ ]  2018-05-23-react-v-16-4
>   * [ ]  2018-06-07-you-probably-dont-need-derived-state (@maicss) #168
>   * [ ]  2018-08-01-react-v-16-4-2
>   * [ ]  2018-09-10-introducing-the-react-profiler(@Fonkie) #253
>   * [ ]  2018-10-01-create-react-app-v2 (@hstarorg)
>   * [ ]  2018-10-23-react-v-16-6
>   * [ ]  2018-11-13-react-conf-recap (@manglobe) #181
>   * [ ]  2018-11-27-react-16-roadmap (@ghycy15)
>   * [ ]  2018-12-19-react-v-16-7
>   * [ ]  2019-02-06-react-v16.8.0 (@JiyuShao) #171
>   * [ ]  2019-02-23-is-react-translated-yet (@penghk)
> * [ ]  Community

I would like to translate, starting with "Concurrent Mode",  Are you ok?

**Comment by yuqingc** [comment]
> I would like to translate, starting with "Concurrent Mode", if that`s ok

@dylanyg "Concurrent Mode" is already done. https://zh-hans.reactjs.org/docs/concurrent-mode-intro.html

**Comment by LeoooY** [comment]
I would like to translate,  "blog/2015-12-18-react-components-elements-and-instances", if not taken.

**Comment by Yilia-Wish** [comment]
I want to translate 2020-02-26-react-v16.13.0

**Comment by ChengYanJin** [comment]
Hello, I would like to translate `cross-origin-errors.md` if nobody has done it yet.

**Comment by redhat123456** [comment]
|title | Processing	|Status|
|------|------------|------|
|Tanger| 2013-06-02-jsfiddle-integration|Translating|

**Comment by redhat123456** [comment]
|title | Processing	|Status|
|------|------------|------|
|2013-06-02-jsfiddle-integration|@Tanger |Translating|

**Comment by mffy** [comment]
Hello, I would like to translate 2018-12-19-react-v-16-7.

**Comment by ifshizuku** [comment]
Hello, I'd like to translate the new version(beta) docs, where and when should I get start, thanks?

**Comment by liangwuweb** [comment]
Hello, I'd like to translate the 2022-03-29-react-v18 if no one has taken it yet.

**Comment by cw1997** [comment]
Hello, I find the 'zh-Hans' docs  [react-dom-client](https://zh-hans.reactjs.org/docs/react-dom-client.html) still don't have a Chinese edition. I'd like to translate it if nobody is working in this section.

**Comment by liangwuweb** [comment]
@cw1997 I guess this repo is not active. I leave a message in May and haven't gotten a reply yet.

**Comment by cw1997** [comment]
> @cw1997 I guess this repo is not active. I leave a message in May and haven't gotten a reply yet.

I see the commit log in the 'main' branch has been updated recently (https://github.com/reactjs/zh-hans.reactjs.org/commits/main), but nobody responded to this PR, may they have another collaborative approach, do you know it? (⊙_⊙;)

**Comment by liangwuweb** [comment]
Sorry, I don't know.😂

**Comment by jsbugwang** [comment]
Title | Processing | Status
-- | -- | --
How to Upgrade to React 18 | @jsbugwang | Translating

**Comment by cyxofgithub** [comment]
> @cw1997 Sorry, I don't know.😂



> Hello, I find the 'zh-Hans' docs [react-dom-client](https://zh-hans.reactjs.org/docs/react-dom-client.html) still don't have a Chinese edition. I'd like to translate it if nobody is working in this section.



> Hello, I find the 'zh-Hans' docs [react-dom-client](https://zh-hans.reactjs.org/docs/react-dom-client.html) still don't have a Chinese edition. I'd like to translate it if nobody is working in this section.



> Hello, I find the 'zh-Hans' docs [react-dom-client](https://zh-hans.reactjs.org/docs/react-dom-client.html) still don't have a Chinese edition. I'd like to translate it if nobody is working in this section.


I also want to do this job, but no one seems to care about PR

**Comment by favorhau** [comment]
|  Title   | Processing  | Status
|  ----  | ----  | ---- |
| Mixins Considered Harmful  | @favorhau | Translating |

**Comment by awxiaoxian2020** [comment]
> > Hello, I'd like to translate the new version(beta) docs, where and when should I get start, thanks?
> 
> Hello, I'd like to translate the new version(beta) docs, where and when should I get start, thanks?

I think we must handling the existing PR first, Stay tuned!

**Comment by awxiaoxian2020** [comment]
The old website has been archived. And we can start new translations at #602 (the list hasn't been ready yet, but I think we can start if you will) and discuss it at #1075.
